iceoryx_posh/test/moduletests/test_mepoo_chunk_header.cpp

```diff
@@ -101,3 +103,29 @@ TEST_F(ChunkHeader_test, FromPayloadFunctionCalledWithNullptrReturnsNullptr)
 {
     EXPECT_THAT(ChunkHeader::fromPayload(nullptr), Eq(nullptr));
 }
+
+TEST_F(ChunkHeader_test, usedChunkSizeIsSizeOfChunkHeaderWhenPayloadIsZero)
+{
+    ChunkHeader sut;
+    sut.m_chunkSize = 2 * sizeof(ChunkHeader);
+    EXPECT_THAT(sut.usedSizeOfChunk(), Eq(sizeof(ChunkHeader)));
+}
+
+TEST_F(ChunkHeader_test, usedChunkSizeIsSizeOfChunkHeaderPlusOneWhenPayloadIsOne)
+{
+    ChunkHeader sut;
+    sut.m_chunkSize = 2 * sizeof(ChunkHeader);
+    constexpr uint32_t PAYLOAD_SIZE{1U};
+    sut.m_payloadSize = PAYLOAD_SIZE;
+    EXPECT_THAT(sut.usedSizeOfChunk(), Eq(sizeof(ChunkHeader) + PAYLOAD_SIZE));
+}
+
+TEST_F(ChunkHeader_test, usedChunkSizeTerminatesWhenPayloadSizeHasSillyValue)
```

Comment:
One could argue what exactly "Silly" is. Maybe rename the test case and name the variable `sillyPayload`?

Reply:
done